_posts/2015-11-05-drupal-web-mapping.md

```diff
@@ -66,5 +66,5 @@ Open Layers does have a cost of more computational and memory overhead than Leaf
 ## Views GeoJSON + Custom leaflet.js code
 
 This is a sort of "headless Drupal" approach, and it's the one we're using on Savas Labs first D8 site, the [Durham Civil Rights map](https://github.com/savaslabs/durham-civil-rights-map). By using the Views GeoJSON module, you can
-render the output of any view containing location information as a GeoJSON feed (potentially even including exposed filters via the query path). Then, just like any GeoJSON feed, that data can be loaded via AJAX into a Leaflet (or OpenLayers, or other frameworks) map. Savas Labs' [Anne Tomasevich](/company/anne-tomasevich/) has a great write-up on just how to do that [in Drupal 8 specifically](/2015/07/06/map-in-drupal-8.html), and I wrote a post a while ago about [how to map GeoJSON data in Leaflet more generally](/2015/05/18/mapping-geojson.html). This approach is still something we're
+render the output of any view containing location information as a GeoJSON feed (potentially even including exposed filters via the query path). Then, just like any GeoJSON feed, that data can be loaded via AJAX into a Leaflet (or OpenLayers, or other frameworks) map. Savas Labs' [Anne Tomasevich](/company//) has a great write-up on just how to do that [in Drupal 8 specifically](/2015/07/06/map-in-drupal-8.html), and I wrote a post a while ago about [how to map GeoJSON data in Leaflet more generally](/2015/05/18/mapping-geojson.html). This approach is still something we're
```

Comment:
Remove link to Anne per former comment. Also, I'm surprised the `/company//` worked, but I guess it didn't throw a link error. Shruggie.